OpenSSL/SSL.py

```diff
@@ -65,6 +65,7 @@ class _buffer(object):
     pass
 
 OP_SINGLE_DH_USE = _lib.SSL_OP_SINGLE_DH_USE
+OP_SINGLE_ECDH_USE = SSL_OP_SINGLE_ECDH_USE
```

Comment:
This needs `_lib.` before it.

Also, I suspect this attribute doesn't always exist, so it might need a conditional.